src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -713,6 +713,17 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
     }
 
     /**
+     * Create and return and un-saved model instance.
+     *
+     * @param  array  $attributes
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function make(array $attributes = [])
```

Comment:
Oh. Looked too quickly. Thought this was defined on the model.

Of course, since it's on the builder it should not be static.

However, why is it on the builder in the first place?